spec-0000/schedule.md

```diff
@@ -1,98 +1,115 @@
-
-2024 – Quarter 3:
-
-- 21 Jul 2024: drop xarray 2022.6.0 (initially released on Jul 22, 2022)
-- 28 Jul 2024: drop scipy 1.9.0 (initially released on Jul 29, 2022)
-- 05 Sep 2024: drop ipython 8.5.0 (initially released on Sep 06, 2022)
-- 15 Sep 2024: drop matplotlib 3.6.0 (initially released on Sep 16, 2022)
-- 18 Sep 2024: drop pandas 1.5.0 (initially released on Sep 19, 2022)
-- 28 Sep 2024: drop xarray 2022.9.0 (initially released on Sep 29, 2022)
-
-2024 – Quarter 4:
-
-- 03 Oct 2024: drop python 3.10 (initially released on Oct 04, 2021)
-- 12 Oct 2024: drop xarray 2022.10.0 (initially released on Oct 13, 2022)
-- 29 Oct 2024: drop ipython 8.6.0 (initially released on Oct 30, 2022)
-- 03 Nov 2024: drop xarray 2022.11.0 (initially released on Nov 04, 2022)
-- 27 Nov 2024: drop ipython 8.7.0 (initially released on Nov 28, 2022)
-- 01 Dec 2024: drop xarray 2022.12.0 (initially released on Dec 02, 2022)
-- 07 Dec 2024: drop scikit-learn 1.2.0 (initially released on Dec 08, 2022)
-- 17 Dec 2024: drop numpy 1.24.0 (initially released on Dec 18, 2022)
-
-2025 – Quarter 1:
-
-- 02 Jan 2025: drop ipython 8.8.0 (initially released on Jan 03, 2023)
-- 02 Jan 2025: drop scipy 1.10.0 (initially released on Jan 03, 2023)
-- 07 Jan 2025: drop networkx 3.0 (initially released on Jan 08, 2023)
-- 17 Jan 2025: drop xarray 2023.1.0 (initially released on Jan 18, 2023)
-- 26 Jan 2025: drop ipython 8.9.0 (initially released on Jan 27, 2023)
-- 06 Feb 2025: drop xarray 2023.2.0 (initially released on Feb 07, 2023)
-- 09 Feb 2025: drop ipython 8.10.0 (initially released on Feb 10, 2023)
-- 12 Feb 2025: drop matplotlib 3.7.0 (initially released on Feb 13, 2023)
-- 27 Feb 2025: drop ipython 8.11.0 (initially released on Feb 28, 2023)
-- 27 Feb 2025: drop scikit-image 0.20.0 (initially released on Feb 28, 2023)
-- 21 Mar 2025: drop xarray 2023.3.0 (initially released on Mar 22, 2023)
-- 29 Mar 2025: drop ipython 8.12.0 (initially released on Mar 30, 2023)
-
-2025 – Quarter 2:
-
-- 02 Apr 2025: drop pandas 2.0.0 (initially released on Apr 03, 2023)
-- 03 Apr 2025: drop networkx 3.1 (initially released on Apr 04, 2023)
-- 13 Apr 2025: drop xarray 2023.4.0 (initially released on Apr 14, 2023)
-- 27 Apr 2025: drop ipython 8.13.0 (initially released on Apr 28, 2023)
-- 18 May 2025: drop xarray 2023.5.0 (initially released on May 19, 2023)
-- 01 Jun 2025: drop scikit-image 0.21.0 (initially released on Jun 02, 2023)
-- 01 Jun 2025: drop ipython 8.14.0 (initially released on Jun 02, 2023)
-- 16 Jun 2025: drop numpy 1.25.0 (initially released on Jun 17, 2023)
-- 22 Jun 2025: drop xarray 2023.6.0 (initially released on Jun 23, 2023)
-- 24 Jun 2025: drop scipy 1.11.0 (initially released on Jun 25, 2023)
-
-2025 – Quarter 4:
-
-- 23 Oct 2025: drop python 3.11 (initially released on Oct 24, 2022)
-
-2026 – Quarter 4:
-
-- 01 Oct 2026: drop python 3.12 (initially released on Oct 02, 2023)
+#### 2023 - Quarter 4:
+
+Recommend drop support for:
```

Comment:
The rendered version could maybe benefit from a tiny bit of different formatting, either leaving more space between this line and the table itself, or making this line with a different typeface (italic should be enough, no need to make it a heading).

Reply:
Yeah, spacing need css change, as prettify reformat the markdown anyway.

Reply:
what about https://github.com/scientific-python/specs/pull/273#discussion_r1344398340 then?

Reply:
You can put the markdown inside of
```
<!-- prettier-ignore-start -->
```
and
```
<!-- prettier-ignore-end -->
```
to prevent prettier from messing with your formatting.

Feel free to let @alphapapa know if you would like any css updates to the theme. You are also welcome to just make a PR if you prefer.